src/flb_http_client.c

```diff
@@ -1107,6 +1107,43 @@ int flb_http_proxy_auth(struct flb_http_client *c,
     return flb_http_add_auth_header(c, user, passwd, FLB_HTTP_HEADER_PROXY_AUTH);
 }
 
+int flb_http_bearer_auth(struct flb_http_client *c, const char *token)
+{
+    int ret;
+    int len_t;
+    int len_h = 8; // length of "Bearer " plus null terminator
```

Comment:
Could you check comment style ?
https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#single-line-comment


```
/* single line comment */
```